relay-event-normalization/src/event.rs

```diff
@@ -438,7 +424,7 @@ fn normalize_user_geoinfo(geoip_lookup: &GeoIpLookup, user: &mut User) {
 }
 
 fn normalize_logentry(logentry: &mut Annotated<LogEntry>, _meta: &mut Meta) -> ProcessingResult {
-    processor::apply(logentry, crate::normalize::logentry::normalize_logentry)
+    processor::apply(logentry, crate::logentry::normalize_logentry)
```

Comment:
would the precedent be that we would use the namespace->verb instead of verb->noun where applicable ? like here doing logentry::normalize instead of logentry::normalize_logentry?